src/storage/src/hummock/file_cache/meta.rs

```diff
@@ -196,6 +196,11 @@ where
             &self.buffer[Self::slot_info_len() * slot
                 ..Self::slot_info_len() * slot + BlockLoc::encoded_len()],
         );
+
+        // Set slot len = 0 to invalidate slot.
+        (&mut self.buffer[slot * Self::slot_info_len() + 4..slot * Self::slot_info_len() + 8])
```

Comment:
minor: slot len is also check in `open`. we can have a helper function to calculate the location of slot len to make the code less error prone and more readable.